website/content/docs/commands/hcp/connect.mdx

```diff
@@ -21,40 +33,60 @@ access to the requested organization, project, and HCP Vault Dedicated cluster.
 If authentication succeeds, the Vault CLI saves the returned HCP token and HCP
 Vault address in the local cache.
 
-## Examples
+### Limitations and warnings
```

Comment:
```suggestion
```

I ended up deleting the limitations/warnings section when it's not needed since it's a subsection of Description and not a top-level heading